osslsigncode.c

```diff
@@ -4290,11 +4290,11 @@ static int engine_load(GLOBAL_OPTIONS *options)
     const char *id = options->p11engine ? options->p11engine : "pkcs11";
     ENGINE *engine;
 
-    if (strchr(id, '.')) {
-        /* Treat strings with a dot as paths to dynamic engine modules */
+    if (strpbrk(id, "/\\") || id[0] == '.') {
```

Comment:
I either don't understand this change, or I understand it and I disagree with it.  The idea of the original code is to use a dot as an indicator of a file extension, which indicates that the parameter is an engine *file* rather than an engine *name*.  The file may still be in the default directory, so it doesn't have to contain slashes, backslashes, or start with a dot.

Reply:
Also see my https://github.com/mtrojnar/osslsigncode/issues/456#issuecomment-3036450736.  If you think that the current logic of just checking for a dot is insufficient or incorrect, please explain why you think it needs changing before implementing a change.  I could neither find an explanation in #456 nor the description of this PR.

Reply:
Got it, I was wrong here, let’s keep the original logic.